java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -728,4 +732,83 @@ CompletableFuture<Long> zrangestore(
      * }</pre>
      */
     CompletableFuture<Long> zlexcount(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>weightableKeys
+     * </code>.<br>
+     * To get the elements with their scores, see {@link #zunionWithScores}.<br>
+     *
+     * @apiNote When in cluster mode, all <code>keys</code> must map to the same <code>hash slot
+     *     </code>.
+     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
+     * @param weightableKeys The keys of the sorted sets. The format for the keys can be:
+     *     <ul>
+     *       <li>When only keys are required, use {@link KeyArray}.
```

Comment:
reword:
> `when uniting sorted sets with equal weights, use {@link KeyArray}`

and so on

Reply:
These weights are 'boost' values, and multiplied by the given sorted set.  Do we want to describe the purpose of the weights here or just refer to the redis/valkey documentation?

Reply:
I can make this more complicated but I fell like weightedKeys have all the necessary documentation but I can copy it over.

Reply:
For weighted keys as score multipliers, use (I'll have this concise description)